style-guide/components/01-components/utilities/spacing/spacing.config.json

```diff
@@ -21,6 +21,13 @@
       "context": {
         "modifier": "standard-gap--bottom"
       }
+    },
+    {
+      "name": "Variable gap - mobile tablet desktop",
+      "notes": "For differing gaps between devices",
```

Comment:
We should update the notes to include:

- Which breakpoints are supported.
- Which size variants are supported (including value used for each size variant)